pkg/providers/tinkerbell/stack/testdata/expected_upgrade.yaml

```diff
@@ -40,6 +40,8 @@ stack:
   loadBalancerIP: 1.2.3.4
   relay:
     enabled: false
+    image: {}
+    initImage: {}
```

Comment:
Can we add the images for these to the test bundle so the test verifies the actual values?